ibm/service/kubernetes/resource_ibm_container_vpc_cluster.go

```diff
@@ -1349,7 +1349,7 @@ func WaitForVpcClusterWokersVersionUpdate(d *schema.ResourceData, meta interface
 	}
 
 	log.Printf("Waiting for worker (%s) version to be updated.", workerID)
-	clusterID := d.Id()
+	clusterID := d.Get("name").(string)
```

Comment:
this changes are not acceptable..why is it changed from id to name

Reply:
Reverted changes